tests/test_safety.py

```
@@ -21,17 +21,25 @@
-from check_photo_backups import (
-    InvHashConfig,
-    SAFE_SAFETY_VALUES,
-    classify_safe_to_delete,
-    iter_inventory_rows,
-    parse_config_str,
-    resolve_inventory_display_path,
-)
+    HAS_CHECK_PHOTO = True
+except ImportError:
+    HAS_CHECK_PHOTO = False
+    InvHashConfig = lambda *a, **k: None
```

Ruff (0.15.10)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `InvHashConfig` as a `def`

(E731)